ipc/shmem_ipc/serverkill.c

```diff
@@ -0,0 +1,29 @@
+#include "common.h"
+
+void server_kill(workspace_t * workspace){
+    
+    for (int i = 0; i < MAX_JOB_BUFFERS; i++){
+		job_buffer_t * current = &(workspace->job_buffers[i]);
+		current->status = SERVER_KILL_SIGNAL;
+	}
+
+
+    return;
+}
+
+int main(){
```

Comment:
We should figure out a way to kill the server without an external program doing it